scripts/SANS/sans/gui_logic/gui_common.py

```diff
@@ -81,7 +87,7 @@ def get_reduction_mode_from_gui_selection(gui_selection):
         return ISISReductionMode.Merged
     elif gui_selection == ALL:
         return ISISReductionMode.All
-    elif gui_selection == SANS2D_LAB or gui_selection == LOQ_LAB or gui_selection == LARMOR_LAB or gui_selection == DEFAULT_LAB:  # noqa
+    elif gui_selection == SANS2D_LAB or gui_selection == LOQ_LAB or gui_selection == LARMOR_LAB or gui_selection == ZOOM_LAB or gui_selection == DEFAULT_LAB:  # noqa
```

Comment:
should the comment be at the end of this line?

Reply:
Thanks for testing :+1:.  The `#noqa` should be on the same line as the thing that causes the flake issue you want to disable, see https://stackoverflow.com/questions/5033727/how-do-i-get-pyflakes-to-ignore-a-statement?answertab=active#tab-top In this case I want everything in one line for increased readability.

Reply:
ok its good to go then

Reply:
Thanks :+1: